packages/cli-kit/src/public/node/api/partners.ts

```diff
@@ -57,12 +57,11 @@ export interface FunctionUploadUrlGenerateResponse {
  * @returns The response of the query.
  */
 export async function getFunctionUploadUrl(token: string): Promise<FunctionUploadUrlGenerateResponse> {
-  const functionUploadUrlGenerateMutation = FunctionUploadUrlGenerateMutation
   const res: FunctionUploadUrlGenerateResponse = await partnersRequest(FunctionUploadUrlGenerateMutation, token)
   return res
 }
 
-const FunctionUploadUrlGenerateMutation = gql`
+export const FunctionUploadUrlGenerateMutation = gql`
```

Comment:
Not a comment on this PR, just generally: should this be in cli-kit at all?

Reply:
Maybe not... But we are also exposing other Partners functions, so not sure. We can review this later when we finish the refactor.